src/geoserverexplorer/test/testplugin.py

```diff
@@ -87,7 +93,8 @@ def functionalTests():
 
     dragdropTest = Test("Verify dragging browser element into workspace")
     dragdropTest.addStep("Setting up catalog and explorer", _setUpCatalogAndExplorer)
-    dragdropTest.addStep("Drag layer from browser into testing workspace of testing catalog")
+    dragdropTest.addStep("Setting up test data project", _loadTestData)
```

Comment:
There is no need to load the data in this case. We are draggin for the browser, so the project data wont be used at all